### PR TITLE
March 2024 update

### PR DESCRIPTION
**Pull request by ashelkovnykov**
I think it speaks for itself

@sigilante @bonbud-macryg @midden-fabler @mopfel-winrux @brbenji

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,269 @@
...
 
-- [urbit-static-page](https://github.com/matildepark/urbit-static-page/) – Mount an arbitrary HTML page to an Urbit ship
-- [create-landscape-app](https://github.com/urbit/create-landscape-app) – Get started making a simple app for Landscape with a few commands
-- [urbit-chatbot-framework](https://github.com/robkorn/urbit-chatbot-framework) – A framework that allows anyone to create an Urbit Chatbot with only a few lines of code
+- [gin & tonic](https://github.com/arthyn/gin-tonic): 2 utilities for making SAIL development easier: auto-sync and auto-refresh
+- [Mast](https://github.com/R-JG/mast): A library for building fully dynamic SAIL frontends
+- [Rudder](https://github.com/Fang-/suite): A framework for SAIL development (Hoon-only HTML)
 
-### Integrations
+## Educational Resources
```
Comment (bonbud-macryg) [inline README.md:168]: Seems like an oversight not to include Hoon Academy and App Academy from urbit.org here.

File: README.md [hunk 2]
```diff
@@ -1,72 +1,269 @@
...
+- [urcrypt](https://github.com/urbit/urcrypt): Cryptography routines used by Urbit jets
+- [Vere](https://github.com/urbit/vere): The original Urbit virtual machine, written in C
+    - Dockerhub images available [here](https://hub.docker.com/r/tloncorp/vere)
+
+## Podcasts
+
+### Series
+
+- Zero K: The Urbit Foundation's official podcast - [YouTube](https://www.youtube.com/@zero-k-podcast?themeRefresh=1)
+- Red Horizon Podcast: The official podcast of ChorusOne's Urbit wing, Red Horizon - [YouTube](https://www.youtube.com/@RedHorizonHost)
```
Comment (bonbud-macryg) [inline README.md]: I think "Chorus One" is two words.

**Review by bonbud-macryg — COMMENTED** [review]
Two nitpicks; otherwise will merge in the next week if @sigilante has no further comments.

File: README.md (hunk 1 above)
Comment (bonbud-macryg) [inline README.md:168]: There's an updated Hoon Cheat Sheet here: https://storage.googleapis.com/media.urbit.org/docs/hoon-cheat-sheets-2023-01-10.pdf

Comment (ashelkovnykov) [reply README.md:168]: I was operating under the impression that this page is for stuff not in the official docs. Do other "awesome" projects usually link to stuff that would be equivalent to Hoon / App Academy? I thought it was more like, "look what people have done on their own!"

**Comment by sigilante** [comment]
https://github.com/0x70b1a5/kenning

**Comment by ashelkovnykov** [comment]
> https://github.com/0x70b1a5/kenning

What is it and is it hosted anywhere currently?

**Comment by sigilante** [comment]
SRRS for longer texts.

"Memorize passages of text. Every passage is called a "kenning"."

I don't believe it's hosted on an active ship right now.

**Comment by ashelkovnykov** [comment]
@sigilante Added `kenning` to "Archive", and `%ahoy` and `%fount` to "Applications"

**Pull request merged**